.circleci/config.yml

```diff
@@ -450,7 +450,7 @@ workflows:
       - Mirror Bugzilla issues into GitHub
     triggers:
       - schedule:
-          cron: "0 * * * *"
+          cron: "0,30 * * * *"
```

Comment:
I hope this syntax is supported!

Reply:
The CircleCI docs helpfully said "look at the manpage for crontab for the syntax" so fingers crossed...